src/routes/Account/containers/AccountContainer.scss

```diff
@@ -1,25 +1,43 @@
-.container {
-  display: flex;
-  flex-direction: row;
+.container, .form, .pane, .settings {
   align-items: center;
+  display: flex;
```

Comment:
Took note of this in Pivotal -- https://www.pivotaltracker.com/story/show/141478437